src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Collections/SeqModule.fs

```diff
@@ -19,11 +19,11 @@ Make sure each method works on:
 * Null Seq    (null)
 *)
 
-[<TestFixture>]
-type SeqModule() =
+[<Parallelizable>][<Category "Collections Tests">][<Category "Seq Tests">]
+module SeqModuleTests =
```

Comment:
this was a basic usability change so that it;d be easy to run the necessary tests in the VS Test Runner, as they are currently setup it's hard to select the right ones. We can remove that commit later if Don doesn't like it, but for now it's useful